Source/Model/ZMManagedObject+Internal.h

```diff
@@ -57,8 +57,6 @@ extern NSString * _Nonnull const ZMManagedObjectLocallyModifiedKeysKey;
 /// The order in which objects are updated to / from the backend. ZMSyncOperationSet uses this.
 + (nullable NSArray <NSSortDescriptor *> *)sortDescriptorsForUpdating;
 + (nullable NSPredicate *)predicateForFilteringResults;
-+ (nullable NSFetchRequest *)sortedFetchRequest;
-+ (nullable NSFetchRequest *)sortedFetchRequestWithPredicate:(nonnull NSPredicate *)predicate;
 + (nullable NSFetchRequest *)sortedFetchRequestWithPredicateFormat:(nonnull NSString *)format, ...;
```

Comment:
Let's make this one `nonnull` for completeness?

```suggestion
+ (nonnull NSFetchRequest *)sortedFetchRequestWithPredicateFormat:(nonnull NSString *)format, ...;
```